src/icons/index.js

```diff
@@ -23,6 +23,7 @@ const icons = {
   search: require('./Search').default,
   smartCollect: require('./SmartCollect').default,
   subscription: require('./Subscription').default,
+  successSnackbar: require('./SuccessSnackbar').default,
```

Comment:
Icons are not specific to features. Why snackbar in name?

Reply:
Also, add the other icon for failure.

Reply:
Sure.